resources/views/home.blade.php

```diff
@@ -21,6 +21,15 @@
                 </div>
             </div>
         </div>
+<div id='map' style='width: 400px; height: 300px;'></div>
+<script>
```

Comment:
Line 25 to 31 should be removed because we already have them in a separate JavaScript file.